testsuite/integration/basic/pom.xml

```diff
@@ -1219,6 +1258,38 @@
                                 <phase>none</phase>
                             </execution>
 
+                            <!-- Tests against the install with JSF support -->
+                            <execution>
+                                <id>jsf-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <jboss.install.dir>${basedir}/target/wildfly-jsf</jboss.install.dir>
+                                        <jboss.home>${project.build.directory}/wildfly-jsf</jboss.home>
+                                        <jboss.home.dir>${project.build.directory}/wildfly-jsf</jboss.home.dir>
+                                        <jbossas.dist>${project.build.directory}/wildfly-jsf</jbossas.dist>
+                                        <jboss.dist>${project.build.directory}/wildfly-jsf</jboss.dist>
+                                        <!-- Override the standard module path that points at the shared module set from dist -->
+                                        <module.path>${project.build.directory}/wildfly-jsf/modules${path.separator}${basedir}/target/modules</module.path>
+                                    </systemPropertyVariables>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${project.build.directory}/wildfly-jsf</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <redirectTestOutputToFile>${testLogToFile}</redirectTestOutputToFile>
+                                    <includes>
+                                        <include>org/jboss/as/test/integration/jsf/**/*TestCase.java</include>
```

Comment:
@yersan I had to add the following in order to execute the test **InjectionBeanToPhaseListenerTest** mentioned in the TP:
`<include>org/jboss/as/test/integration/jsf/phaselistener/injectiontarget/bean/InjectionBeanToPhaseListenerTest.java</include>`
is it actually missing or the executuion is linked somewhere else?

Reply:
Good catch!, that's an issue with this topic branch.

Reply:
I've just fixed this adding explicitly this extension to include the missing test.